.github/workflows/e2e-spin-sleep.yml

```diff
@@ -0,0 +1,184 @@
+name: Spin-Sleep End-to-End Tests
+
+on:
+  workflow_dispatch:
+    schedule:
+      - cron: "0 9 * * 1"
+  push:
+    branches: [main]
+    paths:
+      - "benchmarks/sleeping/**"
+      - "utils/**"
+      - "tools/**"
+      - "runner/**"
+
+  pull_request:
+    branches: [main]
+    paths:
+      - "benchmarks/sleeping/**"
+      - "utils/**"
+      - "tools/**"
+      - "runner/**"
+
+env:
+  GOOS: linux
+  GO111MODULE: on
+  PORT: 50051
+  PLATFORMS: linux/amd64,linux/arm64
+
+jobs:
+  build-and-push:
+    name: Build and push all images
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        service:
+          [
+            sleeping-go
```

Comment:
```suggestion
            sleeping-go,
            spinning-go
```

Reply:
Please add the spinning benchmark otherwise it will never be tested

You also need a second parameter for the docker file:
Similar to:
```
matrix:
        include:
          - service: sleeping-go
            file: benchmarks/sleeping/docker/Dockerfile
```